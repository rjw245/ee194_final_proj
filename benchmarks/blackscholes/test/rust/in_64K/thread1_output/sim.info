{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread1_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../../rust/bs',
             '1',
             '../../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (12850.881369,
            5.9171,
            126192,
            0,
            0,
            0,
            380855,
            0,
            0,
            0,
            12032,
            0,
            0,
            0,
            241008,
            799861),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/8_cores/thread1_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../../rust/bs 1 ../../../../inputs/in_64K.txt'],
 't_elapsed': 16283.21413397789,
 't_start': 1460988456.246169,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 3451822080L}
