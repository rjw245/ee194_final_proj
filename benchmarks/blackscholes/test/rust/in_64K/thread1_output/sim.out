                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |  4326690385 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  Cycles                             |  2552846954 |  2552846954 |  2552846954 |  2552846954 |  2552846954 |  2552846954 |  2552846954 |  2552846954
  IPC                                |        1.69 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  Time (ns)                          |   959716900 |   959716900 |   959716900 |   959716900 |   959716900 |   959716900 |   959716900 |   959716900
  Idle time (ns)                     |        4013 |   959716900 |   959716900 |   959716900 |   959716900 |   959716900 |   959716900 |   959716900
  Idle time (%)                      |        0.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0%
Branch predictor stats               |             |             |             |             |             |             |             |            
  num correct                        |   151452601 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num incorrect                      |     6725015 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  misprediction rate                 |       4.25% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
  mpki                               |        1.55 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
TLB Summary                          |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   875448484 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num misses                       |         188 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |  2875368490 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num misses                       |       40373 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.01 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  L2 TLB                             |             |             |             |             |             |             |             |            
    num accesses                     |       40561 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num misses                       |        1100 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       2.71% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
Cache Summary                        |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |            
    num cache accesses               |   875448484 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |    20495714 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       2.34% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        4.74 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
  Cache L1-D                         |             |             |             |             |             |             |             |            
    num cache accesses               |  2875368995 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |      178040 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.01% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.04 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
  Cache L2                           |             |             |             |             |             |             |             |            
    num cache accesses               |    20673867 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |       65829 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.32% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.02 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
  Cache L3                           |             |             |             |             |             |             |             |            
    num cache accesses               |       65949 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |       33724 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |      51.14% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.01 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
DRAM summary                         |             |             |             |             |             |             |             |            
  num dram accesses                  |       16861 |           0 |           0 |           0 |       16858 |           0 |           0 |           0
  average dram access latency (ns)   |       55.01 |         inf |         inf |         inf |       54.96 |         inf |         inf |         inf
  average dram queueing delay        |        1.58 |        0.00 |        0.00 |        0.00 |        1.53 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |            
  num loads from dram                |        4123 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
